test/test_log_to_cerr/test_log_to_cerr.cpp

```diff
@@ -15,7 +15,7 @@ int main(void)
     log.warning("Testing warning");
     log.error("Testing error");
     log.fault("Testing fault");
-    log.failur("Testing failur");
+    log.failure("Testing failur");
```

Comment:
`log.failure("Testing failure");`
a small correction of the spelling mistake in the string.